src/app/domain/case-viewer/components/summary-panel/summary-panel.component.html

```diff
@@ -1,7 +1,14 @@
 <div data-selector="summary-panel-component" class="govuk-grid-row">
-    <ng-container *ngFor="let section of panelData.sections">
-        <div class="govuk-grid-column-one-half">
-            <app-data-list [title]="section.name" [dataList]="section.fields"></app-data-list>
-        </div>
-    </ng-container>
+    <div class="govuk-grid-column-two-thirds">
+        <ng-container *ngFor="let section of panelData.sections">
+            <div  class="govuk-grid-row">
+                <div class="govuk-grid-column" *ngIf="section.id !== 'recentEvents'">
+                    <app-data-list [title]="section.name" [dataList]="section.fields"></app-data-list>
```

Comment:
This feels pretty hacky. Do the recent events need to be in the template? Would they need to be configurable per jurisdiction or will they always be in the summary? If they will always be there then just hard code it.

Reply:
Be nice to encapsulate this as a function inside the actual component. That way we can write tests against it rather  than putting a lot of logic into the view

Reply:
couldn't think an easy solution to change this up would be nice to set position of the sub types 

because it like
| ==== 2/3 ==== | == 1/3 == |
| =Summary=     |  timeline  |
| =Repr...         = |

for example on a mobile I bet you they'd want it like

| =Summary=     | 
| =Repr...         = |
|  timeline           |